main.js

```diff
@@ -257,3 +253,57 @@ function winnerNameOutput() {
     winnerName.innerText = twoName.value;
   }
 }
+
+// function below will reset game to default with new generated correct number
+document.addEventListener('keyup', enableReset);
+resetButton.addEventListener('click', resetGame);
+
+function enableReset() {
+  var hasMin = minRange.value !== "";
+  var hasMax = maxRange.value !== "";
+  var hasOneName = oneName.value !== "";
+  var hasTwoName = twoName.value !== "";
+  var hasOneGuess = oneGuess.value !== "";
+  var hasTwoGuess = twoGuess.value !== "";
+  var isFilled = hasMin || hasMax || hasOneName || hasTwoName || hasOneGuess || hasTwoGuess;
+  if (isFilled) {
+    resetButton.disabled = false;
+  } else {
+    resetButton.disabled = true;
+  }
+}
+
+function resetGame() {
+  defaultSetRange();
+  clearAllField();
+  defaultLastetGuess();
```

Comment:
there is a typo in this function and the one defining it.  should read defaultLatestGuess instead.

Reply:
I will change the typo to the correct spelling

Reply:
Thanks!